app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -657,3 +665,20 @@ DartdocReport _emptyReport() => DartdocReport(
       // TODO: add meaningful message for missing documentation on dartdoc
       documentationSection: null,
     );
+
+/// Creates a report section when running dartdoc failed to produce content.
+ReportSection _dartdocFailedSection(String abortMessage) {
```

Comment:
What if we did this on the pana-side?

Reply:
The original method `dartdocFailedSection` was/is in `pana`, but we don't use it inside `pana`. I used to think that this has a better place in `pana`, but only if we were to use it there too.

Reply:
ah, nvm... this is because when running pana we don't know if running dartdoc has failed.